extensions/2.0/Khronos/KHR_texture_video/KHR_texture_video.md

```diff
@@ -0,0 +1,155 @@
+# KHR_texture_video
+
+## Contributors
+
+* Dwight Rodgers, Adobe
+* Gary Hsu, Microsoft
+* Emmett Lalish, Google
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension allows glTF models to use videos as textures. A client that does not implement this extension can ignore the provided video and continue to rely on the PNG and JPG textures available in the base specification. Defining a fallback texture is optional. 
+
+Video textures support use cases such as water or other textures that continuously repeat, coordination of video texture playback by animation (in conjunction with `KHR_animation_pointer`), or control of video texture playback by interactivity (in conjunction with `KHR_interactivity`).
+
+## glTF Schema Updates
+
+The `KHR_texture_video` extension is added to the `textures` node and specifies a `source` property that points to the index of the `images` node which in turn points to the video.
```

Comment:
> points to the index of the `images` node which in turn points to the video.

How does `images` in turn point to a video? In your example, the entirety of the `"images"` only contains one object with a reference to a PNG file:

```
"images": [
    {
        "uri": "image.png"
    }
]
```

I think what you meant is that `"source": 0,` inside of `"KHR_texture_video": {` points to `/extensions/KHR_texture_video/videos/0`, but this is not at all clear from the document.

Reply:
Thanks for catching this. I have updated this section to be more clear and precise in the latest commit. Please let me know if it is clear and unambiguous now.